specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2021-07-01/examples/Updates/HealthCheck.json

```diff
@@ -0,0 +1,43 @@
+{
+  "parameters": {
+    "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "resourceGroupName": "System.Redmond",
+    "updateLocation": "Redmond",
+    "updateName": "Microsoft1.0.180302.1",
+    "api-version": "2021-07-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/3c2a4c80-e594-46eb-b923-98fd98d50155/resourcegroups/System.Redmond/providers/Microsoft.Update.Admin/updateLocations/redmond/updates/Microsoft1.0.180410.1/updateRuns/5495b71f-692c-4580-8dcd-03dc6ce00209",
+        "name": "5495b71f-692c-4580-8dcd-03dc6ce00209",
+        "type": "Microsoft.Update.Admin/updateLocations/updates/updateRuns",
+        "location": "redmond",
+        "tags": {},
+        "properties": {
+          "progress": {
+            "name": "HealthChecking"
+          },
+          "duration": "PT0S",
```

Comment:
Not sure how `PT0S` translates to a duration.  Assume something like `17S` would make more sense as if PT means "pacific time" there is no concept of timezone in a duration value.

Reply:
This ISO 8601 format: https://en.wikipedia.org/wiki/ISO_8601#Durations

Reply:
Makes sense... thanks!